README.md

```diff
@@ -65,6 +65,22 @@ Additional data:
 * `branch`: The name of the branch, if a branch was pushed.
 * `tag`: The name of the tag, if a tag was pushed.
 
+#### Errors
+
+Emitted if there was an error handling the request or processing the payload data.
+
+```js
+notifier.on( "error", function( error ) {
+	console.error( error.message );
+})
+```
+
+##### `Invalid data parsing payload`
+
+Additional error attribute:
+
+* `payload` (Object): The raw payload containing the data from the webhook.
```

Comment:
Let's just reduce this whole section to:

If there is an error parsing the payload, the `error` object will contain a `payload` property which contains the raw payload that was delivered.